docs/source/release/v6.1.0/mantidworkbench.rst

```diff
@@ -8,52 +8,64 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 
+General
+#######
 - Added Floating/On Top setting for all the windows that are opened by workbench (plots, interfaces, etc.)
```

Comment:
I think this is one of the big Core changes, and it's worth users knowing that they can chose this behaviour of interfaces. It's particularly important on say Ubuntu, where a user may choose `Floating` to allow the minimise/maximise buttons. This is all explained [here](https://docs.mantidproject.org/nightly/workbench/windowbehaviour.html) so please add a link to it with 
```
ref:`Whatever words you want<WorkbenchWindowBehavior>`
```
We could maybe even make a GIF related to this (they work on the forum, maybe on the docs?) or just an image of the new setting that can be changed. Feel free to poke me about this on Monday!